src/components/comment/Comment.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import './Comment.scss';
+
+export function PostsComments({comment}) {
+    if (comment.length === 0) return null;
+    return (
+        comment.map(item => {
+            return (
+                <div className='post-comment'>
```

Comment:
I suppose here also should be `key` added

Reply:
I added key.